rakefile.rb

```diff
@@ -33,16 +33,14 @@
   #run nuget to get linux packages
   nuget_restore linux_distro
 
+  # Tar file produced by the script
   tar_file_name = "ospsuite_#{product_version}.tar.gz"
   
-  # Tar file produced by the script
-  tar_file = File.join(build_dir, tar_file_name)
-
   #unzip it in a temp folder
   temp_distro_dir = File.join(temp_dir, linux_distro)
   FileUtils.mkdir_p temp_distro_dir
 
-  command_line = %W[xzf #{tar_file} -C #{temp_distro_dir}]
+  command_line = %W[xzf #{tar_file_name} -C #{temp_distro_dir}]

```

Comment:
@msevestre so finally it worked after I removed the concatenation with the **build_dir**
(the current folder of the build process is apparently already the build_dir).

So somehow `File.join(build_dir, tar_file_name)` destroyed the path by removing the slashes from the **build_dir**.
Cannot explain it other than a bug in rake(?)

Reply:
no it did not fully work... the linux binaries are not under artifacts anymore, rrrr

Reply:
>a bug in rake(?)

Lol no way . fIle.join is exactly made for this and rake has been doing this forever without problems. So unless a weird version of ruby and rake is used, I can't understand why it stopped working suddently

Reply:
I want to try to reproduce locally. What does build_dir look like?

Reply:
build_dir is something like C:/projects/ospsuite-r
and it's the root of the cloned repor; ospsuite_x.y.zip is created also directly in the build_dir